File: Version3.0/SimiCartPluginFW/SimiFramework/CustomClass/SimiResponder.m
//
//  SimiResponder.m
//  SimiCartPluginFW
//
//  Created by Tan Hoang on 2/7/14.
//  Copyright (c) 2014 SimiTeam. All rights reserved.
//

#import "SimiResponder.h"

@implementation SimiResponder

@synthesize status, message, error, other;

- (id)init
{
    self = [super init];
    if (self) {
        status = @"";
        message = nil;
        error = nil;
    }
    return self;
}

- (NSString *)responseMessage{
    return self.message;
}

@end

File: simitracking/simitracking/SimiFramework/CustomClass/SimiResponder.swift
//
//  SimiResponder.swift
//  simitracking
//
//  Created by Hai Nguyen on 11/8/16.
//  Copyright © 2016 SimiCart. All rights reserved.
//

import UIKit

class SimiResponder: NSObject {
    var status:String = ""
    var message:Array<NSObject>!
    var other:Array<NSObject>!
    var error:Error!
    
    func responseMessage()->String{
        var mess:String = ""
        if (error == nil) {
            if (message != nil && message.count >= 1) {
                mess = message[0] as! String
            }
        }
        return mess
    }
            
}
